now.json

```diff
@@ -1,8 +1,7 @@
 {
   "build": {
     "env": {
-      "GITHUB_CLIENT_ID": "@github_client_id",
-      "GITHUB_CLIENT_SECRET": "@github_client_secret"
+      "GITHUB_TOKEN": "@github_token"
```

Comment:
I don't think it's possible to use a client id and secret with the graphql api so this needs to be updated to a [github token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).